Source/Core/Core/ConfigManager.h

```diff
@@ -316,6 +318,8 @@ struct SConfig
   bool m_SSLVerifyCert;
   bool m_SSLDumpRootCA;
   bool m_SSLDumpPeerCert;
+  std::unique_ptr<Core::NetworkCaptureLogger> m_network_logger{
```

Comment:
Should this not be lazy initialized? It seems kind of wasteful to allocate by default for this when the majority of users likely won't even need to use it.